py_mypy/cli/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+py_binary(
+    name = "cli",
+    main = "__main__.py",
+    srcs = ["__main__.py"],
+    deps = [
+        "//py_mypy/order_processing",
+        "//py_mypy/inventory_management",
+        "@pip//click",
+    ],
+    tags = [
+        "mypy",
```

Comment:
if a user isn't paying attention, are they going to assume they need to put "mypy" tags all over their codebase?